docs/webiny-apps/page-builder/development/plugins-reference/app.md

```diff
@@ -0,0 +1,185 @@
+---
+id: app
+title: App Plugins Reference
+sidebar_label: App
+---
+
+# Summary
+
+[`pb-editor-page-element`](/docs/webiny-apps/page-builder/development/plugins-reference/app#pb-editor-page-element)
+
+[`pb-editor-page-element-advanced-settings`](/docs/webiny-apps/page-builder/development/plugins-reference/app#pb-editor-page-element-advanced-settings)
+
+
+
+---
+
+### [`pb-editor-page-element`](/docs/webiny-apps/page-builder/development/plugins-reference/app#pb-editor-page-element)
+
+#### Summary
+
+Enables adding custom element plugin in the Editor
```

Comment:
`Enables adding custom element plugin in the Editor` => `Enables adding new page elements to the page editor.`